x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -168,188 +217,390 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
+                        />
+                      ) : null
+                    }
+                  >
+                    <EuiFieldNumber
+                      isInvalid={!!errors.min_number_of_allocations}
+                      min={1}
+                      value={config.adaptive_allocations?.min_number_of_allocations}
+                      onChange={(event) => {
+                        onConfigChange({
+                          ...config,
+                          adaptive_allocations: {
+                            enabled: config.adaptive_allocations?.enabled ?? false,
+                            ...config.adaptive_allocations,
+                            min_number_of_allocations: Number(event.target.value),
+                          },
+                        });
+                      }}
+                    />
+                  </EuiFormRow>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiFormRow
+                    label={
+                      <FormattedMessage
+                        id="xpack.ml.trainedModels.modelsList.startDeployment.maxAllocationsLabel"
+                        defaultMessage="Max"
+                      />
+                    }
+                    isInvalid={!!errors.max_number_of_allocations}
+                  >
+                    <EuiFieldNumber
+                      isInvalid={!!errors.max_number_of_allocations}
+                      min={(config.adaptive_allocations?.min_number_of_allocations ?? 1) + 1}
+                      value={config.adaptive_allocations?.max_number_of_allocations}
+                      onChange={(event) => {
+                        onConfigChange({
+                          ...config,
+                          adaptive_allocations: {
+                            enabled: config.adaptive_allocations?.enabled ?? false,
+                            ...config.adaptive_allocations,
+                            max_number_of_allocations: Number(event.target.value),
+                          },
+                        });
+                      }}
+                    />
+                  </EuiFormRow>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiDescribedFormGroup>
+          ) : null}
+
+          {config.priority !== undefined ? (
+            <EuiDescribedFormGroup
+              titleSize={'xxs'}
+              title={
+                <h3>
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.modelsList.startDeployment.priorityLabel"
+                    defaultMessage="Priority"
+                  />
+                </h3>
+              }
+              description={
+                <FormattedMessage
+                  id="xpack.ml.trainedModels.modelsList.startDeployment.priorityHelp"
+                  defaultMessage="Select low priority for demonstrations where each model will be very lightly used."
```

Comment:
I leave some alternatives:

```suggestion
                  defaultMessage="Select low priority for demonstrations where models will be used minimally."
```
or
```suggestion
                  defaultMessage="Select low priority for demonstrations with minimal model usage."
```

Reply:
FYI this text was already in place